SerialPrograms/Source/PokemonLA/Programs/ShinyHunting/PokemonLA_BurmyFinder.cpp

```diff
@@ -134,7 +134,7 @@ std::unique_ptr<StatsTracker> BurmyFinder::make_stats() const{
 
 
 struct BurmyFinder::TreeCounter{
-    std::array<uint64_t, 10> tree = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+    std::array<uint64_t, 16> tree = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
std::array<uint64_t, 16> tree = {}
or
std::array<uint64_t, 16> tree = { 0 }
is enough to init them all to 0 ;)

Reply:
Removed all of them :) 
Thanks!